README.md

```diff
@@ -341,7 +341,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
 * [bqplot](https://github.com/bloomberg/bqplot) - Interactive Plotting Library for the Jupyter Notebook
 * [Dash](https://plot.ly/products/dash/) - Built on top of Flask, React and Plotly aimed at analytical web applications.
     * [awesome-dash](https://github.com/Acrotrend/awesome-dash)
-* [ggplot](https://github.com/yhat/ggpy) - Same API as ggplot2 for R.
+* [plotnine](https://github.com/has2k1/plotnine) - Same API as ggplot2 for R.
```

Comment:
Hello @JarnoRFB,

It's probably worth to update the description as well.

```suggestion
* [plotnine](https://github.com/has2k1/plotnine) - A grammar of graphics for Python.
```

This suggestion is based on plotnine's description on GitHub and PyPI.

Reply:
I would probably keep the reference to `ggplot2` as lack of it is often one of the main complaints of people coming form R, so I would hope that more people could find it here. See updated PR for suggestion.